hw/dv/tools/xcelium/cov_report.tcl

```diff
@@ -20,12 +20,35 @@ set dut_instance [string trim $::env(dut_instance)]
 # Load the merged coverage database.
```

Comment:
Do we need tcl files for other tools?

Reply:
@matutem most tools use tcl scripts (vcs/dc/vivado ...) 
I am not sure that answers your question. but We do have other tcl scripts in place

Reply:
For VCS, we are able to make do with external commands / command line switches (example: `urg`).

Reply:
Please resolve